samples: bluetooth: ccp_call_control_client: correct the README

aaronyegx opened the pull request:
The README of ccp_call_control_client sample describled the name and behavior incorrectly. This commit corrects it.

Thalley reviewed (dismissed):
Thanks :)

kartben reviewed (commented):
Would be nice to align the wording of the sample description with guidelines ("word this as if you were completing the sentence "This code sample shows how to ...", i.e. it shouldn't say "sample" :)
https://docs.zephyrproject.org/latest/samples/index.html#bluetooth

kartben commented on samples/bluetooth/ccp_call_control_client/README.rst:
```diff
@@ -1,9 +1,9 @@
 
-   CCP Call Control Server sample that registers one or more TBS bearers and advertises the
-   TBS UUID(s).
+   CCP Call Control Client sample that scans for the CCP servers to connect and discovers the
+   TBS and GTBS.
```
Yes, something along these lines is what I meant - starting with a verb. Maybe the below (still a bit long but I guess it's difficult to make it much shorter)

> Implement a CCP Call Control Client that scans for CCP servers, connects to them, and discovers the TBS and GTBS.

aaronyegx committed:
samples: bluetooth: ccp_call_control_client: correct the README

The README of ccp_call_control_client sample describled the name and
behavior incorrectly. This commit corrects it.

Signed-off-by: Aaron Ye <aye@ambiq.com>

Thalley commented on samples/bluetooth/ccp_call_control_client/README.rst:
```diff
@@ -1,9 +1,9 @@
-   CCP Call Control Server sample that registers one or more TBS bearers and advertises the
-   TBS UUID(s).
+   Implement a CCP Call Control Client that scans for CCP servers, connects to them, and discovers
+   the TBS and GTBS. Note that its functionality will change over time as the CCP API progresses.
```
Hmm, yeah, at least the `Note that its functionality will change over time as the CCP API progresses.` should not be part of this. 

Could also just be 

```c
Implement a CCP Call Control Client.
```

If we want it to be very brief

aaronyegx replied:
Can the description just describe the current implementation of sample? We really don't know how the module/subsystem will grow/change, using a very brief description may make the README unclear IMO.

kartben replied:
I think you might be overthinking this :) The description is meant to really have the main keywords that one might use when browsing the list of samples / using the search feature. The actual, detailed, descripiton & instructions belong to the actual README imo
